packages/patternfly-4/react-core/src/demos/BulkSelectTable.md

```diff
@@ -0,0 +1,228 @@
+---
+title: 'Bulk select table'
+section: 'demos'
+---
+
+## Bulk select table demo
+
+import { Pagination, PaginationVariant, Title, Checkbox } from '@patternfly/react-core';
+import { Table, TableHeader, TableBody} from '@patternfly/react-table';
+
+```js
+import React from 'react';
+import { Pagination, PaginationVariant, Title, Checkbox } from '@patternfly/react-core';
```

Comment:
Can probably remove `PaginationVariant` from the imports

Reply:
You are right, good catch!